src/plugins/intel_gna/src/transformations/matops_to_dwsc.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ov {
+namespace intel_gna {
+namespace pass {
```

Comment:
I don't see any registration of the passes to pass manager.
It should be added in following file:
https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gna/src/gna_transformations_pipeline.cpp

This would allow to check also if there is no regressions with existing implementation.

Reply:
Received information that backend is not yet ready. closing comment